File: php nvoice/Faculteit-fe499af0-daf42fa6/faculteit.php
<?php

echo "van welk getal wil je de faculteit weten?" . PHP_EOL;
$x = readline("");

$y = 1;

for ($i = 1; $i <= $x; $i++) {
    $y = $y * $i ;
}

echo "faculteit van $x is $y";


?>

File: Python Novice/Faculty-cbbe2fc3-8ccaa020/faculteit.py
x = int(input("Van welk getal wil je de faculteit weten? "))
total = 1
for i in range(1, x + 1):
    total *= i
print("De faculteit van ", x, " is ", total)

#################### andere manier

def factorial(n):
    if n == 0:
        return 1
    return n * factorial(n-1)


number = input("Enter a number: ")

number = int(number)


result = factorial(number)
print("The factorial of", number, "is", result)
